Bottom: 33a90000c9ebee9ca9ac3177177feb81f6941f6a
Top:    82d055e73b7f44fdbe4a05b3ef99474a1c4eff84
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-27 14:13:15 +0800

Refresh of autofs-5.0.7-dont-prode-rdma-mounts.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index b644071..e3d7746 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -137,8 +137,8 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 				flags &= ~MOUNT_FLAG_USE_WEIGHT_ONLY;
 			} else if (strncmp("use-weight-only", cp, o_len) == 0) {
 				flags |= MOUNT_FLAG_USE_WEIGHT_ONLY;
-			} else if (strcmp("proto=rdma", cp, o_len) == 0 ||
-				   strcmp("rdma", cp, o_len) == 0) {
+			} else if (strncmp("proto=rdma", cp, o_len) == 0 ||
+				   strncmp("rdma", cp, o_len) == 0) {
 				rdma = 1;
 			} else {
 				if (strncmp("vers=4", cp, o_len) == 0 ||
